core/src/main/java/org/infinispan/manager/EmbeddedCacheManager.java

```diff
@@ -239,6 +232,8 @@
     * @param configurationName name of the configuration template to use
     * @return null if no configuration exists as per rules set above, otherwise
     *         returns a cache instance identified by cacheName
+    * @deprecated Use {@link EmbeddedCacheManager#defineConfiguration(String, String, Configuration)} and
+    * {@link EmbeddedCacheManager#getCache(String)} instead
```

Comment:
I usually add a version number to make it easier to decide when to actually remove the deprecated classes/methods.